lib/rmt/cli/mirror.rb

```diff
@@ -155,4 +197,17 @@ def finish_execution
       raise RMT::CLI::Error.new('The command exited with errors.') unless options[:do_not_raise_unpublished] && in_alpha_or_beta?
     end
   end
+
+  def files_size_format(downloaded_files_size)
+    (downloaded_files_size / 1024000.0).floor(2)
```

Comment:
Here, you could use Rails [`number_to_human_size`](https://api.rubyonrails.org/classes/ActiveSupport/NumberHelper.html#method-i-number_to_human_size), it nicely formats the size.